index.html

```diff
@@ -123,6 +123,14 @@ <h2>The Club</h2>
           <td><a href="https://github.com/shasankp000/PyCraft">PyCraft</a></td>
           <td>a minecraft launcher built in python, for fun</td>
         </tr>
+        <tr>
+          <td><a href="https://github.com/burntcarrot/hashsearch">hashsearch</a></td>
+          <td>a reverse image search engine using perceptual hashes, for fun</td>
```

Comment:
"reverse image search engine using perceptual hashes" is 7 words, can you shorten to 5 or less?

Reply:
sure!